### DISCUSSION TITLE
Welcome to awesome-datascience Discussions!

### DISCUSSION THREAD

**Discussion by hmert** [academic/awesome-datascience#228] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AliHaider20** [comment]
First of all great initiative @hmert. Hi everybody. I have recently completed my bachelors in computer science and now started contributing to open source. My major is AI.

**Reply by nonsonebo** [reply]
please how many courses are in data science and machine learning?

**Reply by AliHaider20** [reply]
Data science is a field, not a program. You need to start with the basics and then advance. Follow https://i.am.ai/roadmap/#note roadmap.

**Reply by scottedwards2000** [reply]
nice resource @AliHaider20 - thanks for sharing!

**Comment by afsvieira** [comment]
Thank you @hmert to shared this awesome knowledge with us!
